src/scss/components/_homepage.scss

```diff
@@ -0,0 +1,118 @@
+body {
```

Comment:
In general the amount of custom styles you are using indicate that you could be getting more out of BS